### ISSUE TITLE
HTML date input and SC 3.3.2

### ISSUE THREAD

**Issue by pete-murphy** [w3c/wcag#4205]
I'm trying to understand the wording of SC 3.3.2 in WCAG 2.2:

> A field for entering a date has text instructions to indicate the correct format for the date.

This changed from WCAG 2.0 (emphasis added):

> A field for entering a date _contains initial text_ which indicates the correct format for the date.

Would normal usage of `input[type="date"]` fail 3.3.2 if the date format is not also included in the label or somewhere else that's persistently shown?

**Comment by patrickhlauke** [comment]
for `<input type="date">` it's the browser/UA's responsibility to somehow convey the additional instruction (and the whole point is that, in theory at least, the browser will use the user's locale to adapt it to their needs, so an author wouldn't know in advance what the format that the field expects is, unless they implemented their own locale-sniffing). long story short, no, i would not fail `<input type="date">` without additional instructions
